components/ome-xml/pom.xml

```diff
@@ -46,6 +46,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
By the way, is there any way of having a PR in both the regular merge build and the breaking build at the same time?  This one is now no longer breaking and should be fine to move over to the regular build, as are some of the others.